src/python_testing/TC_ACL_2_8.py

```diff
@@ -187,18 +259,28 @@ async def test_TC_ACL_2_8(self):
             "Targets should be NullValue")
         asserts.assert_equal(entry.fabricIndex, f1)
 
-        for entry in acl_list:
+        for entry in acl_list_filtered:
             asserts.assert_not_equal(
                 entry.fabricIndex, f2, "Should not contain entry with FabricIndex F2")
 
+        # Read with fabric_filtered=False
+        acl_list_unfiltered = await self.read_single_attribute_check_success(
+            dev_ctrl=self.th1, endpoint=0, cluster=ac_cluster, attribute=acl_attr,
+            fabric_filtered=False
+        )
+        logging.info("TH1 read ACL result (fabric_filtered=False): %s", str(acl_list_unfiltered))
+        asserts.assert_equal(len(acl_list_unfiltered), 2, "Should have two ACL entries when not fabric filtered")
+
```

Comment:
Can you please add a check here that the data for the non-accessing fabric is empty. You should get back a response, but it won't have any fields, and the client will fill in the defaults, which is empty everything. 

Please also add a comment as to why you're checking for empty everything.

Reply:
and wherever else this happens.